src/GitVersionExe/GitVersionRunner.cs

```diff
@@ -10,24 +10,23 @@
 
 namespace GitVersion
 {
-    public class GitVersionApplication : IGitVersionApplication
+    public class GitVersionRunner : IGitVersionRunner
```

Comment:
This feels more like an `Execute` sort of class than the ones that are currently named `Execute`, since this deals with command line arguments and such.

Reply:
Changed to `GitVersionExecutor`